### PR TITLE
Internet connections over mobile data & WLAN

### PR DESCRIPTION
**Pull request by jovirkku**
Signed-off-by: Jorma Virkkunen <jorma.virkkunen@jolla.com>

### CODE REVIEW

**Review by jpwalden — APPROVED** [review]
LGTM, however couldn't quite get the "(NOTE LINK)" -text on the line 27

**Comment by vigejolla** [comment]
> LGTM, however couldn't quite get the "(NOTE LINK)" -text on the line 27

I have a feeling it's a reminder @jovirkku wrote for himself to add a link to the correct chapter there -> I'm not merging this quite yet.

**Comment by jpwalden** [comment]
> > LGTM, however couldn't quite get the "(NOTE LINK)" -text on the line 27
> 
> I have a feeling it's a reminder @jovirkku wrote for himself to add a link to the correct chapter there -> I'm not merging this quite yet.

Yep, just discussed that with Jovirkku -> you are right, he will fix that.

**Comment by jovirkku** [comment]
> LGTM, however couldn't quite get the "(NOTE LINK)" -text on the line 27

I'll fix it. My bad.

File: Support/Help_Articles/Internet_Connections/README.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+
+Sailfish devices prefer WLAN to mobile networks by default.  If a suitable and known WLAN network (used previously by your phone) is available, Sailfish will create a connection via it automatically, as soon as the network is observed to be in the range (this is a distance of up to 20 meters typically) and measured to have a strong enough signal level.
+
+If there are no WLAN networks in the range, or if they have not been previously used by this particular device, then your Sailfish device will create the Internet connection via a mobile network - given that you have a contract for data transfer with the relevant operator (carrier) and that you are in the range (i.e., the signal level is good enough).
+
+The smooth switchover from WLAN to a mobile network and vice versa naturally requires that both of those network types are enabled on your device.
+
+It is possible to affect the behaviour of your phone by configuring the Internet connections to your liking. Read more on this in chapter "_Configuring Internet connections in Settings_" below.
```
Comment (vigejolla) [inline Support/Help_Articles/Internet_Connections/README.md:27]:
```suggestion
It is possible to affect the behaviour of your phone by configuring the Internet connections to your liking. Read more on this in chapter [Configuring Internet connections in Settings](#configuring-internet-connections-in-settings) below.
```

**Comment by vigejolla** [comment]
You could still squash those two commits

**Comment by jovirkku** [comment]
> You could still squash those two commits

OK. I thought this "edit at web UI" would do everything for me. But no.

**Comment by vigejolla** [comment]
> OK. I thought this "edit at web UI" would do everything for me. But no.

Yup, it's really good but not perfect.

**Review by vigejolla — APPROVED** [review]

**Pull request merged**